vector/src/main/res/values/strings.xml

```diff
@@ -2566,8 +2572,20 @@
     <string name="auth_pin_reset_title">Reset pin</string>
     <string name="auth_pin_new_pin_action">New pin</string>
     <string name="auth_pin_reset_content">To reset your PIN, you\'ll need to re-login and create a new one.</string>
+    <string name="settings_security_application_protection_title">Protect access to the application.</string>
```

Comment:
**Suggestion 1 (preferred)**
Protect access ~to the application.~

_I personally don't think you need to say the application, given the context of being in the app. The more text, the more a user has to read to understand and I haven't see other applications using this phrasing._

**Suggestion 2**
Protect access to Element ~to the application.~

**Suggestion 3**
Protect access to the application ~remove full stop at the end.~
_There's no punctuation in other titles_

Reply:
I take suggestion 1, thanks